### PR TITLE
DON'T MERGE!

### PR DESCRIPTION
**Pull request by kabir**
‼️ Don't merge this ‼️ 

It is a sample PR for trying out the deptree reporter

### CODE REVIEW

File: build-legacy/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <parent>
         <groupId>org.wildfly.core</groupId>
         <artifactId>wildfly-core-parent</artifactId>
-        <version>9.0.0.Beta2-SNAPSHOT</version>
+        <version>9.0.0.Beta3-SNAPSHOT</version>
```
Comment (jamezp) [inline build-legacy/pom.xml]: This is just a test comment on a specific line.

**Comment by kabir** [comment]
Normal comment

**Comment by kabir** [comment]
**Dependency Tree Analyzer Output:**

**Comment by kabir** [comment]
**Dependency Tree Analyzer Output:**

New Dependencies:
* io.undertow:undertow-coreNEW:jar:2.0.18.Final:compile

Removed Dependencies:
* org.projectodd.vdx:vdx-wildfly:jar:1.1.6:compile

Major Version Changes:
* javax.json:javax.json-api:jar:1.1.2:compile -> 2.1.2
* org.projectodd.vdx:vdx-core:jar:1.1.6:compile -> 2.1.6

**Comment by kabir** [comment]
Retest this please

**Pull request closed without merging**